docs/source/chaincode4ade.rst

```diff
@@ -159,102 +159,102 @@ in the ``CreateAsset`` function below.
+        return fmt.Errorf("the asset %s already exists", id)
+      }
 
-        return ctx.GetStub().PutState(id, assetJSON)
+      asset := Asset{
+        ID:             id,
+        Color:          color,
+        Size:           size,
+        Owner:          owner,
+        AppraisedValue: appraisedValue,
+      }
+      assetJSON, err := json.Marshal(asset)
+      if err != nil {
+        return err
       }
 
+      return ctx.GetStub().PutState(id, assetJSON)
+    }
+
 Now that we have populated the ledger with some initial assets and created an asset,
 let's write a function ``ReadAsset`` that allows us to read an asset from the ledger.
 
 .. code:: go
 
   // ReadAsset returns the asset stored in the world state with given id.
-     func (s *SmartContract) ReadAsset(ctx contractapi.TransactionContextInterface, id string) (*Asset, error) {
-      assetJSON, err := ctx.GetStub().GetState(id)
-      if err != nil {
-        return nil, fmt.Errorf("failed to read from world state: %v", err)
-      }
-      if assetJSON == nil {
-        return nil, fmt.Errorf("the asset %s does not exist", id)
-      }
-
-      var asset Asset
-      err = json.Unmarshal(assetJSON, &asset)
-      if err != nil {
-        return nil, err
-      }
+  func (s *SmartContract) ReadAsset(ctx contractapi.TransactionContextInterface, id string) (*Asset, error) {
+    assetJSON, err := ctx.GetStub().GetState(id)
+    if err != nil {
+      return nil, fmt.Errorf("failed to read from world state: %v", err)
+    }
+    if assetJSON == nil {
+      return nil, fmt.Errorf("the asset %s does not exist", id)
+    }
 
-      return &asset, nil
+    var asset Asset
+    err = json.Unmarshal(assetJSON, &asset)
+    if err != nil {
+      return nil, err
     }
 
+    return &asset, nil
+  }
+
 Now that we have assets on our ledger we can interact with, let's write a chaincode function
 ``UpdateAsset`` that allows us to update attributes of the asset that we are allowed to change.
 
 .. code:: go
 
   // UpdateAsset updates an existing asset in the world state with provided parameters.
-     func (s *SmartContract) UpdateAsset(ctx contractapi.TransactionContextInterface, id string, color string, size int, owner string, appraisedValue int) error {
-        exists, err := s.AssetExists(ctx, id)
-        if err != nil {
-          return err
-        }
-        if !exists {
-          return fmt.Errorf("the asset %s does not exist", id)
-        }
-
-        // overwriting original asset with new asset
-        asset := Asset{
-          ID:             id,
-          Color:          color,
-          Size:           size,
-          Owner:          owner,
-          AppraisedValue: appraisedValue,
-        }
-        assetJSON, err := json.Marshal(asset)
-        if err != nil {
-          return err
-        }
+  func (s *SmartContract) UpdateAsset(ctx contractapi.TransactionContextInterface, id string, color string, size int, owner string, appraisedValue int) error {
```

Comment:
The color formatting did not return for this function for some reason. All other functions look good.

Reply:
I dont really understand, what you mean?

Reply:
There's no color formatting in the `UpdateAsset` function in the built docs: https://hyperledger-fabric--1850.org.readthedocs.build/en/1850/chaincode4ade.html#initializing-the-chaincode (same for `DeleteAsset` and a couple other places). It looks like on master this was the case as well so it's probably not a huge deal but odd why it's not formatting on read-the-docs.